backend/middleware/exception.py

```diff
@@ -68,13 +68,17 @@ def process_exception(
         # Handle only when running in production, for debug mode
         # Django takes care of displaying a detailed HTML page.
         if not settings.DEBUG and exception:
+            logger.error(f"Unhandled exception by DRF for {request}, logging error...")
+            ExceptionLoggingMiddleware.format_exc_and_log(
+                request=request, exception=exception
+            )
             detail = {"detail": "Error processing the request."}
             return HttpResponse(json.dumps(detail), status=500)
         return None
 
     @staticmethod
     def format_exc_and_log(
-        request: Request, response: Optional[Response], exception: Exception
+        request: Request, exception: Exception, response: Optional[Response] = None
```

Comment:
@chandrasekharan-zipstack won't changing this order have impact on existing usage?

Reply:
In the [existing usage](https://github.com/Zipstack/unstract/blob/0012d7e111953446ab6ab4fbafb481a5479e5a1b/backend/middleware/exception.py#L35-L37) we pass it as kwargs so it remains unaffected. I've checked for other usages as well